metaflow/plugins/argo/argo_workflows.py

```diff
@@ -1916,6 +1931,107 @@ def _slack_success_template(self):
             Http("POST").url(self.notify_slack_webhook_url).body(json.dumps(payload))
         )
 
+    def _heartbeat_daemon_template(self):
+        # Use any step that is known to exist, as we only care
+        # about being able to retrieve the code package for the utility scripts.
+        executable = self.environment.executable("_parameters")
+        run_id_template = "argo-{{workflow.name}}"
+        entrypoint = [executable, "-m metaflow.plugins.argo.heartbeat_daemon "]
```

Comment:
Minor nit - maybe just metaflow.plugins.argo.daemon - it is likely that we would want to use this container for more work besides heartbeats in the near future